packages/commonwealth/client/scripts/views/components/MarkdownEditor/toolbars/Select.tsx

```diff
@@ -0,0 +1,98 @@
+import React, { useEffect, useRef, useState } from 'react';
+
+type Option = {
+  value: string;
+  label: string;
+};
+
+type SelectProps = {
+  options: Option[];
+  onChange: (value: string) => void;
+  placeholder?: string;
+};
+
+const Select: React.FC<SelectProps> = ({ options, onChange, placeholder }) => {
```

Comment:
1. we don't use React.FC anywhere, let's stay consistent.
2. could you change the name of the component to be more specific. We already have generic Select. 
3. do we even need this? Can't you reuse select we already have?
4. another consistency issue, in some components you destructure props and in some you don't. Pick one style and follow it please